frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -29,7 +39,48 @@ class BrfssProvider extends VariableProvider {
     df = this.filterByGeo(df, breakdowns);
     df = this.renameGeoColumns(df, breakdowns);
 
+    // TODO How to handle territories?
+    let acsBreakdowns = breakdowns.copy();
+    acsBreakdowns.time = false;
+
+    let consumedDatasetIds = ["brfss"];
+
     if (breakdowns.geography === "national") {
+      // Because we add together the estimated state totals
```

Comment:
IMO a more descriptive comment on how the logic for this case works would be useful (e.g. BRFSS is a survey, sampling rates are diff across states, so we have to compute estimated state totals first)

Reply:
Hundo p